libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/thrift/ThriftValidate.java

```diff
@@ -269,6 +270,9 @@ private static void validateLicenseIdMatch(License license) throws SW360Exceptio
     public static void prepareProject(Project project) throws SW360Exception {
         assertNotEmpty(project.getName());
         project.setType(TYPE_PROJECT);
+        if(!project.isSetClearingState()) {
+           project.setClearingState(ProjectClearingState.OPEN);
```

Comment:
We indent with multiples of 4 spaces:
```suggestion
            project.setClearingState(ProjectClearingState.OPEN);
```

Reply:
Sorry for that.Done with the changes.